<launch>
	<!-- Hold head in place and reset odometry -->
	<node pkg="teach_repeat" type="miro_setup.py" name="miro_setup">
		<rosparam param="lift">30</rosparam>
		<rosparam param="yaw">-15</rosparam>
		<rosparam param="pitch">0</rosparam>
	</node>

	<!-- Uncompress synced messages and publish info -->
	<node pkg="teach_repeat" type="camera_calibrator.py" name="miro_camera_calibrator" output="screen" >
		<rosparam param="calibration_file_left" subst_value="True">$(find teach_repeat)/calibration/left_240.yaml</rosparam>
		<rosparam param="calibration_file_right" subst_value="True">$(find teach_repeat)/calibration/right_240.yaml</rosparam>
	</node>

</launch>
